src/account/useAuthentication.js

```diff
@@ -5,31 +5,32 @@ import { getAuth, onAuthStateChanged} from "firebase/auth";
 // TODO: Add SDKs for Firebase products that you want to use
 // https://firebase.google.com/docs/web/setup#available-libraries
 
-// Your app's Firebase configuration (HARDCODED. NEEDS Revision)
-const firebaseConfig = {
-    apiKey: "PLACEHOLDER",
-    authDomain: "PLACEHOLDER",
-    projectId: "PLACEHOLDER",
-    storageBucket: "PLACEHOLDER",
-    messagingSenderId: "PLACEHOLDER",
-    appId: "PLACEHOLDER"
+// Your app's Firebase configuration (HARDCODED. NEEDS REVISION)
+// Left below as a placeholder.
+  const firebaseConfig = {
+    apiKey: process.env.FIREBASE_API_KEY,
```

Comment:
Awesome to use `process.env`.
Note: In a Create React App project, environment variables need to start with `REACT_APP_` to be embedded into the (Cloudflare Pages) build.
Please change to:
```
const firebaseConfig = {
  apiKey: process.env.REACT_APP_FIREBASE_API_KEY,
  authDomain: process.env.REACT_APP_FIREBASE_AUTH_DOMAIN,
  projectId: process.env.REACT_APP_FIREBASE_PROJECT_ID,
  storageBucket: process.env.REACT_APP_FIREBASE_STORAGE_BUCKET,
  messagingSenderId: process.env.REACT_APP_FIREBASE_MESSAGING_SENDER_ID,
  appId: process.env.REACT_APP_FIREBASE_APP_ID
};
```
Please delete `databaseURL` because we use MongoDB instead of the firebase database.

Reply:
Will change and push. thanks!

Reply:
Also, you should request to merge your branch into `dev` instead of `main` branch. Please check the `CONTRIBUTING.md` file.

Reply:
> Also, you should request to merge your branch into `dev` instead of `main` branch. Please check the `CONTRIBUTING.md` file.

@markquark125 You don't need to create a new pull request to change to merge your `#account` branch into the `dev` branch. Please click the `Edit` button on the top-right to change the base branch from `main` to `dev`.

Reply:
Edited the pull request so #account -> dev